policy/scan/local_scanner.go

```diff
@@ -568,6 +569,34 @@ func (s *localAssetScanner) run() (*AssetReport, error) {
 	return ar, nil
 }
 
+func noPolicyErr(availablePolicies []string, filter []string) error {
+	var sb strings.Builder
+	sb.WriteString("bundle doesn't contain any policies\n")
+	sb.WriteString("\n")
+
+	if len(availablePolicies) > 0 {
+		sb.WriteString("The following policies are available:\n")
+		for i := range availablePolicies {
+			policyMrn := availablePolicies[i]
+			sb.WriteString("- " + policyMrn)
```

Comment:
just double checking that you didn't want a newline here

Reply:
Great catch, I'll add the newline there